### ISSUE TITLE
Package tests fail with `TypeError: iopackContent.common.titleLang is not iterable`

### ISSUE THREAD

**Issue by l3d00m** [ioBroker/testing#395]
After a fresh (typescript or javascript) adapter creation with `npx @iobroker/create-adapter`, the testing fails with a TypeError.

`npm run test:package` fails with the following output:

```
npm run test:package

> iobroker.auto_strom_ts@0.0.1 test:package
> mocha test/package --exit



  Validate the package files
    Ensure they are readable
      package.json
        ✓ exists
        ✓ contains valid JSON
        ✓ is an object
      io-package.json
        ✓ exists
        ✓ contains valid JSON
        ✓ is an object
    Check contents of package.json
      ✓ The property "name" exists
      ✓ The property "version" exists
      ✓ The property "description" exists
      ✓ The property "author" exists
      ✓ The property "license" exists
      ✓ The property "main" exists
      ✓ The property "repository" exists
      ✓ The property "repository.type" exists
      ✓ The package name is correct
      ✓ The repository type is "git"
    Check contents of io-package.json
      ✓ The property "common.name" exists
      ✓ The property "common.titleLang" exists
      ✓ The property "common.version" exists
      ✓ The property "common.news" exists
      ✓ The property "common.desc" exists
      ✓ The property "common.icon" exists
      ✓ The property "common.extIcon" exists
      ✓ The property "common.license" exists
      ✓ The property "common.type" exists
      ✓ The property "common.authors" exists
      ✓ The property "native" exists
      ✓ The title does not contain "adapter" or "iobroker"
      ✓ titleLang is an object to support multiple languages
      1) titleLang does not contain "adapter" or "iobroker"
      ✓ The description is an object to support multiple languages
      ✓ common.authors is an array that is not empty
      ✓ common.news is an object that contains maximum 20 entries
      ✓ Materialize is supported
    Compare contents of package.json and io-package.json
      ✓ The name matches
      ✓ The version matches
      ✓ The license matches


  36 passing (43ms)
  1 failing

  1) Validate the package files
       Check contents of io-package.json
         titleLang does not contain "adapter" or "iobroker":
     TypeError: iopackContent.common.titleLang is not iterable
      at Context.<anonymous> (node_modules/@iobroker/testing/build/tests/packageFiles/index.js:143:58)
      at processImmediate (internal/timers.js:456:21)



npm ERR! code 1
npm ERR! path /home/thomas/projects/ioBroker.auto_strom_ts
npm ERR! command failed
npm ERR! command sh -c mocha test/package --exit

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/thomas/.npm/_logs/2021-03-13T21_11_43_889Z-debug.log
```

maybe related to #385 ?

**Comment by AlCalzone** [comment]
Fixed in v2.4.4 (available in a couple of minutes). Thanks!

**Comment by l3d00m** [comment]
That was fast, thanks for the fix!

(And also many thanks for maintaining all the other projects, the iobroker create adapter tool is great and so is zwave-js)
